shell/platform/embedder/embedder.h

```diff
@@ -1133,6 +1209,9 @@ extern const int32_t kFlutterSemanticsCustomActionIdBatchEnd;
 /// Action overrides are custom actions that the application developer requests
 /// to be used in place of the standard actions in the `FlutterSemanticsAction`
 /// enum.
+///
+/// @deprecated     Use `FlutterSemanticsCustomAction2` instead. Adding members
+///                 to this struct breaks the ABI of `FlutterUpdateSemantics`.
```

Comment:
Similar comment to above:
```suggestion
/// @deprecated     Use `FlutterSemanticsCustomAction2` instead. In order to
///                 preserve ABI compatility for existing users, no new fields  
///                 will be added to this struct. New fields will continue to         
///                 be added to `FlutterSemanticsCustomAction2`.
```

Reply:
Fixed